@uportal/waffle-menu/src/WaffleMenu.js

```diff
@@ -44,12 +44,11 @@ const WaffleDropdown = styled.ul`
   color: #212529;
   text-align: left;
   list-style: none;
-  background-color: #fff;
   background-clip: padding-box;
   border: 1px solid rgba(0, 0, 0, 0.05);
   width: 220px;
   flex-flow: row wrap;
-  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
+  box-shadow: 2px 2px 5px #999;
```

Comment:
This is an overriding change from my merge this morning.

Reply:
Yeah sorry, the post commit hooks changed it for me and I left it...